functional-test-app/grails-app/services/com/testacl/ReportService.groovy

```diff
@@ -17,13 +18,13 @@ class ReportService {
 
 	@PreAuthorize('hasPermission(#report, admin)')
 	@Transactional
-	void addPermission(Report report, String username, int permission) {
+	void addPermission(@P("report") Report report, String username, int permission) {
```

Comment:
@jamesfredley  @jdaugherty  @codeconsole  As mentioned in the other comment, here are the test changes where the `@P` annotation was added to get parameter name discovery working again.